packages/cli/src/__tests__/e2e/jobrunner-test-cases.ts

```diff
@@ -16,6 +16,7 @@ export const testCases: WorkflowTestCase[] = [
     workflow: {
       name: "simpleCalculator",
       format: "0.1",
+      __type: "PolywrapWorkflow",
```

Comment:
Now that I am seeing this, should we call this manifest `PolywrapTest`? :thinking: I remember you had some thoughts regarding separation of concerns when it comes to `test` and `workflow` but not sure

Just bringing it up in case it is important, feel free to ignore this comment :smile:

Reply:
Agree but will be done in separate PR